### PR TITLE
Fix trailing whitespace in debian templates

### PR DESCRIPTION
**Pull request by avtobiff**
Remove trailing whitespace in templates: Lintian complains.

P: node-agent-base source: file-contains-trailing-whitespace debian/changelog (line 6)
P: node-agent-base source: file-contains-trailing-whitespace debian/rules (line 16)
P: node-agent-base source: file-contains-trailing-whitespace debian/rules (line 17)

### CODE REVIEW

File: npm2deb/templates.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
   * Initial release (Closes: #nnnn)
 
  -- %(debian_author)s  %(date)s
```
Comment (shanavas786) [inline npm2deb/templates.py]: isn't here still a newline ?

Comment (pravi) [reply npm2deb/templates.py]: That is a required blank line if you meant line 4 here.

Comment (shanavas786) [reply npm2deb/templates.py]: No, I meant after line 5

Comment (avtobiff) [reply npm2deb/templates.py]: Ending file with newline is ok, empty lines are not.

Before this change there was a line 6 in debian/changelog.

Comment (shanavas786) [reply npm2deb/templates.py]: got it.

**Pull request merged**